documentation/dev-portal/src/community-resources/community-applications-and-guides.md

```diff
@@ -55,7 +55,53 @@ We love seeing our developer community create applications using Nym. If you wou
 
 <!--NymDrive *end-->
 
+<!--NymDashboard *start-->
 
+## <img src='../images/profile_picture/nym_dashboard_pp.svg' style="float: right; width: 75px; height: 75px;">Nym Dashboard
```

Comment:
Do they not use the same shield logo as below for the dashboard? nw if they do just checking

Reply:
resolved this